### PR TITLE
Add missing Marathon props and improve networks description.

### PR DESCRIPTION
**Pull request by ghost**
## Description
Added missing props. Updated for Marathon 1.5 networks.

## Urgency
- [ ] Blocker 
- [ x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, and 1.10).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.10/deploying-services/marathon-parameters.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ The container information.
 - **portMappings** The port mappings between host and container.  A port mapping is a tuple that contains a host port (`hostPort`), container port (`containerPort`), service port (`servicePort`), and protocol (`protocol`).  Port mappings are similar to passing `-p` into the Docker command line to specify a relationship between a port on the host machine and a port inside the container. 
 
     - **containerPort** The container port (e.g., `8080`).
-    - **hostPort** The host port (e.g., `0`). The default value is `0`. In `USER` mode, the hostPort is not required, but if left unspecified Marathon will not randomly allocate a port.
```
Comment (sascala) [inline 1.10/deploying-services/marathon-parameters.md]: do you mean the `networks.container` parameter?

File: 1.10/deploying-services/marathon-parameters.md [hunk 2]
```diff
@@ -53,36 +53,34 @@ The container information.
     - **forcePullImage** Whether to pull the image, regardless if it is already available on the local system.
     - **image** The path to the Docker image.
     - **privileged** Whether to give extended privileges to this container. For more information, see the [Docker run command](https://docs.docker.com/engine/reference/commandline/run/).
-    
       - `"privileged": false` Do not give extended privileges. This is the default value.
       - `"privileged": true` Give extended privileges. 
     - **parameters** Command-line options for the `docker run` command executed by the Mesos containerizer. Parameters passed in this manner are not guaranteed to be supported in the future, as Mesos may not always interact with Docker via the CLI.
-    - **pullConfig** A secret whose value is a stringified JSON object in a Secret Store. 
+    - **pullConfig** A secret whose value is a stringified JSON object in a Secret Store. See [](/docs/1.10/deploying-services/private-docker-registry/#secret-store-instructions)
```
Comment (sascala) [inline 1.10/deploying-services/marathon-parameters.md]: missing link text + period.

**Review by sascala — APPROVED** [review]
small edits, otherwise 👍

File: 1.10/deploying-services/marathon-parameters.md (hunk 1 above)
Comment (stbof) [reply 1.10/deploying-services/marathon-parameters.md]: Not sure what you are referring to in that comment. Also, there is no `networks.container` parameter, only `networks.mode`, which can be set to `container` or `container/bridge`.

Comment (sascala) [reply 1.10/deploying-services/marathon-parameters.md]: OK, NM. I found "networks" confusing and thought it was a parameter maybe. Maybe remove "networks" to read, "In container mode...."

Comment (stbof) [reply 1.10/deploying-services/marathon-parameters.md]: @sascala `networks` **is** a parameter. It has three subparameters: `mode`, which can be host, container, or container/bridge.

**Pull request merged**